tests/support/activemq/connections_details.py

```diff
@@ -3,12 +3,18 @@
 
 import requests
 
-from django_stomp.helpers import eval_str_as_boolean
+from django.conf import settings
 from parsel import Selector
+
+from django_stomp.helpers import eval_str_as_boolean
 from tests.support.dtos import ConsumerStatus
 
 
-def consumers_details(connection_id, host="localhost") -> Generator[ConsumerStatus, None, None]:
+def consumers_details(
+    connection_id: str,
+    host=settings.STOMP_SERVER_HOST,
+    port: str = settings.STOMP_SERVER_INTERFACE_PORT,
```

Comment:
**suggestion:** Since these params have a default value, the type could be `Optional`.